burst-junit4/src/main/java/com/squareup/burst/BurstRunner.java

```diff
@@ -104,6 +104,6 @@
    */
   private Description describeChildPlain(FrameworkMethod method) {
     return Description.createTestDescription(getTestClass().getJavaClass(),
-        method.getMethod().getName());
+        method.getMethod().getName(), method.getAnnotations());
```

Comment:
👍 seems like the right thing to do even though the code doesn't currently require it.